### ISSUE TITLE
Bootstrap script doesn't get invoked in titan console 

### ISSUE THREAD

**Issue by curtisallen** [thinkaurelius/titan#824]
Given a simple bootstrap script

setup.groovy:

``` groovy
g = new TinkerGraph()
```

When starting the gremlin console you used to be able to pass a bootstrap script. In titan-hadoop 0.5.x this doesn't seem to work. 

``` bash
titan-0.5.0-hadoop1 $ bin/gremlin.sh setup.groovy  
         \,,,/
         (o o)
-----oOOo-(_)-oOOo-----
gremlin> g
No such property: g for class: groovysh_evaluate
Display stack trace? [yN] %
```

Compare with gremlin-groovy

``` bash
gremlin-groovy-2.5.0 $ bin/gremlin.sh setup.groovy

         \,,,/
         (o o)
-----oOOo-(_)-oOOo-----
==>tinkergraph[vertices:0 edges:0]
gremlin> g
==>tinkergraph[vertices:0 edges:0]
gremlin> 
```

I also noticed that if you pass in a path to a script that doesn't exist a FIleNotFoundException is no longer thrown.

**Comment by dalaro** [comment]
This now has a test (60672b030dfac452ac2ae71a3eaf74d627be5318) and a fix (1b23ce606090eb7e9407e63e26bf8e4e2418ac42).  Here's an example post-fix that borrows from the new test:

```
$ cat titan-dist/src/test/resources/test_gremlin_init.groovy
// Used by gremlin-shell-args.expect.vm
test_i = 2 + 2
test_g = new TinkerGraph()
test_hadoop_class = HadoopFactory.class
test_titan_class = TitanFactory.class

$ bin/gremlin.sh titan-dist/src/test/resources/test_gremlin_init.groovy

         \,,,/
         (o o)
-----oOOo-(_)-oOOo-----
==>true
==>4
==>tinkergraph[vertices:0 edges:0]
==>class com.thinkaurelius.titan.hadoop.HadoopFactory
==>class com.thinkaurelius.titan.core.TitanFactory
gremlin> test_i
==>4
gremlin> test_g
==>tinkergraph[vertices:0 edges:0]
gremlin> test_hadoop_class
==>class com.thinkaurelius.titan.hadoop.HadoopFactory
gremlin> test_titan_class
==>class com.thinkaurelius.titan.core.TitanFactory
gremlin> 
```

**Comment by curtisallen** [comment]
:+1:
